patched-vscode/extensions/sagemaker-extension/src/extension.ts

```diff
@@ -17,6 +17,7 @@ import * as console from "console";
 
 
 const PARSE_SAGEMAKER_COOKIE_COMMAND = 'sagemaker.parseCookies';
+const ENABLE_AUTO_UPDATE_COMMAND = 'workbench.extensions.action.enableAutoUpdate';
```

Comment:
nice that we're able to do this directly from the notification. I don't see this command in [this doc](https://code.visualstudio.com/api/references/commands) though. If you have any other reference, would be helpful to link. Even if it's not exposed in their docs, it'll still be helpful to add comment here so that this is something we check when we pull in new version of Code OSS to ensure it still works.

Reply:
Yeah, for this one, I found in https://code.visualstudio.com/docs/editor/extension-marketplace#_extension-autoupdate it says `Use the Enable Auto Update for All Extensions command to re-enable auto update.` - I also couldn't found the command in the Commands Reference

I added a note, that we need to be cautious when pulling in new VSCode version

Reply:
okay, also linking the command mention in Code OSS codebase - [link](https://github.com/microsoft/vscode/blob/4224d78a27f544e30303adc464648403b456f2b0/src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts#L633-L634)